gobblin-compaction/src/main/java/gobblin/compaction/mapreduce/MRCompactorJobRunner.java

```diff
@@ -168,27 +167,12 @@ private void submit(Job job) throws ClassNotFoundException, IOException, Interru
 
   private void moveTmpPathToOutputPath() throws IOException {
     LOG.info(String.format("Moving %s to %s", this.tmpPath, this.outputPath));
+    this.fs.delete(this.outputPath, true);
```

Comment:
But earlier output path was only deleted if the option `compaction.force.reprocess` was true, it requires the user to explicitly ask for output path to be deleted. In this case, the output is deleted without user allowing it.